package.json

```diff
@@ -4,14 +4,14 @@
   "description": "simple controls for an ipfs node",
   "main": "src/index.js",
   "scripts": {
-    "lint": "aegir-lint",
-    "coverage": "aegir-coverage",
-    "test": "aegir-test --env node",
-    "docs": "aegir-docs",
-    "release": "aegir-release --env node --docs",
-    "release-minor": "aegir-release --type minor --env node --docs",
-    "release-major": "aegir-release --type major --env node --docs",
-    "coverage-publish": "aegir-coverage publish"
+    "lint": "aegir lint",
+    "coverage": "aegir coverage",
+    "test": "aegir test --target node",
+    "docs": "aegir docs",
+    "release": "aegir release -t node",
+    "release-minor": "aegir release --type minor -t node",
+    "release-major": "aegir release --type major -t node",
+    "coverage-publish": "aegir coverage -u"
```

Comment:
Please make an aegir upgrade be part of a separate PR

Reply:
kk, will pull aegir out.